doc/content/3.documentation/5.configuration/2.transports/21.azure-functions.md

```diff
@@ -4,6 +4,16 @@ navigation.title: Azure Functions
 
 # Azure Functions Configuration
 
+::alert{type="info"}
+**No Longer Recommended**
```

Comment:
It was never recommended, I'd prefer to phrase this as a superior alternative with a better deployment model and full use of MassTransit.